src/app/page.tsx

```
@@ -371,20 +391,52 @@ export default function Page() {
                         ))}
-                      </ul>
+                      </ul> */}
                     </CardContent>
                   </div>
                 </Card>
-                {!maxTicketsReached && (
+                <RadioGroup defaultValue={ticketSelected as string}>
+                  {EVENT?.tickets?.map((ticket) => (
+                    <>
+                      <Label htmlFor={ticket?.title}>
+                        <Card
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)